README.md

```diff
@@ -262,18 +264,62 @@ func main() {
 }
 ```
 
-## Contribute
+### Send a mail with send API v 3.1
+
+```go
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	mailjet "github.com/mailjet/mailjet-apiv3-go"
+)
+
+func main() {
+	m := mailjet.NewMailjetClient(
+		os.Getenv("MJ_APIKEY_PUBLIC"),
+		os.Getenv("MJ_APIKEY_PRIVATE"))
+
+	messagesInfo := []mailjet.InfoMessagesV31{
+		mailjet.InfoMessagesV31{
+			From: &mailjet.RecipientV31{
```

Comment:
I chose the field name `Recipient` in order to be consistant with the existing code 
The struct definition used for SendAPI InfoSendEmail also use `Recipient` to bundle email and name